fluid/chinese_ner/scripts/train.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+export MKL_NUM_THREADS=1
+export OMP_NUM_THREADS=1
+
+mode=$1 # gpu, cpu, mkldnn
+if [ "$mode" = "CPU" ]; then
+  device="CPU"
+  save_model_dir="cpu_model"
+elif [ "$mode" = "GPU" ]; then
+  device="GPU"
+  save_model_dir="gpu_model"
+elif [ "$mode" = "MKLDNN" ]; then
+  device="CPU"
+  save_model_dir="mkldnn_model"
+  export FLAGS_use_mkldnn=1
```

Comment:
why there is no `--parallel $parallel` in this script like ocr crnn ctc?

Reply:
But MKLDNN couldn't use ParallelExecutor, how should it run MKLDNN?

Reply:
I ran it with MKLDNN flag and it trained the model just fine

Reply:
Thus, what's the speed of MKLDNN+ParallelExecutor? Will it be slower than MKLDNN+Executor?

Reply:
It runs much faster with just an Executor. I'll add an appropriate flag.